docs/core/tools/local-tools-how-to-use.md

```diff
@@ -0,0 +1,191 @@
+---
+title: How to install and use .NET Core local tools
+description: Learn how to install and use a .NET tool as a local tool.
+author: tdykstra
+ms.author: tdykstra
+ms.date: 02/12/2020
+---
+
+# Install and use a .NET Core local tool using the .NET Core CLI
+
+**This article applies to:** ✔️ .NET Core 3.x SDK
+
+This tutorial teaches you how to install and use a local tool. You use a tool that you create in the [first tutorial of this series](global-tools-how-to-create.md).
+
+## Prerequisites
+
+* Complete the [first tutorial of this series](global-tools-how-to-create.md).
+* Install the .NET Core 2.1 runtime.
+
```

Comment:
you also need the 3.x sdk

Reply:
That's covered by the first tutorial -- prerequisite for that one is 3.x.